main/src/ca/uwaterloo/flix/util/ClassList.scala

```diff
@@ -38,6 +38,10 @@ object ClassList {
     *
     * Finally, remove the `./` prefix, e.g., `./com/sun/...` should be `com/sun/...`.
     *
+    * Note that we will remove all classes with a lowercase name.
     */
-  val TheList: List[String] = LocalResource.get("/src/ca/uwaterloo/flix/util/ClassList.txt").split('\n').map(_.trim).toList
+  val TheList: List[String] = {
```

Comment:
I would revert the changes to ClassList.scala.

it should be the completer that filters out the non-capital classes. 

The reason is that for such classes we will eventually need to importer to be even smarter (I will explain later).

So If you could move the check

Reply:
Move into AutoImportCompleter.